docs/src/installation.md

```diff
@@ -105,6 +105,20 @@ scoop install temurin17-jdk
 
 Windows users may also prefer using an [installer](https://adoptium.net/temurin/releases/?version=17&os=windows&arch=x86&package=jdk).
 
+#### Java Versions
+
+Different JVM versions need diffrent Scala versions. And the highest Scala version is set by the compiler plugin for a Chisel version.
+
+current highest supported versions are:
+
+| Chisel | Scala   | Java
+|--------|---------|------
+| 3.5.6  | 2.13.10 | 17
+| 3.6.1  | 2.13.14 | 22
+| 5.3.x  | 2.13.14 | 22
+| 6.5.x  | 2.13.14 | 22
```

Comment:
```suggestion
| Chisel | Scala   | Java
|--------|---------|------
| 3.5.6  | 2.13.10 | 17
| 3.6.1  | 2.13.14 | 22
| 5.3.x  | 2.13.14 | 22
| 6.6.x  | 2.13.16 | 23
| 7.0.0-RC1 | 2.13.16 | 23
```

A few things to consider:
* Adding Chisel 7 since it's in release candidate
* Should we also put minimum Scala and Java versions?
* Should we link to the Scala JDK compatibility page? https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html
* Should we mention that 6.6 and 7.0.0-RC1 (by virtue of supporting Scala 2.13.16) support Java 24 (early access) or wait to mention it until another Scala version comes out that supports it for sure?

Reply:
I would wait on Java 24, as 23 is the actual version.